examples/self_supervised_learning/losses/_wav2vec2_loss.py

```diff
@@ -0,0 +1,66 @@
+from typing import Optional, Tuple
+
+import torch
+import torch.nn.functional as F
+from torch import Tensor
+
+
+def compute_contrastive_loss(x, mask_indices, targets, neg_is_pos, reduce, logit_temp = 0.1):
```

Comment:
could you also add docstring to the function?
```suggestion
def compute_contrastive_loss(x: torch.Tensor, mask_indices: torch.Tensor, targets: torch.Tensor, neg_is_pos: torch.Tensor, reduce: str, logit_temp: int = 0.1):
```